setup.py

```diff
@@ -19,8 +22,7 @@
     license=xs.LICENSE,
     url=xs.HOMEPAGE,
     platforms=xs.PLATFORMS,
-    packages=find_packages(),
-    namespace_packages=['xstatic', 'xstatic.pkg', ],
+    packages=['xstatic.pkg.bootstrap'],
     include_package_data=True,
     zip_safe=False,
     install_requires=[],  # nothing! :)
```

Comment:
this could be changed to require xstatic >= 2.0.0, see https://github.com/xstatic-py/xstatic/issues/15#issuecomment-2810069557 .